MultiTable/src/SubjectGroup.js

```diff
@@ -10,6 +10,25 @@ function ascendingCaseInsensitive(a, b) {
   return d3a.ascending(a.toLowerCase(), b.toLowerCase());
 }
 
+function aspectSorter(a, b) {
```

Comment:
This looks like it sorts _either_ by rank _or_ by name, but if there is a rank, it needs to sort by rank and then _within_ rank, sort by name.